src/App.jsx

```diff
@@ -1,5 +1,23 @@
+import classes from './App.scss';
+
+import Header from './components/header/Header';
+import Side from './components/side/Side';
+import Main from './components/main/Main';
+
+import { BrowserRouter, Route } from 'react-router-dom';
+
 function App() {
-  return <div>Hello world!</div>;
+  return (
+    <BrowserRouter>
+      <div className={`${classes.container} ${classes.container_vertical}`}>
+        <Header />
+        <div className={`${classes.container} ${classes.container_horizontal}`}>
```

Comment:
if container_horizontal and container_vertical are not common classes that can be used by every component, their names should be more specific